charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -277,7 +279,11 @@ scrape_configs:
     replacement: ${1}://${2}${3}
     target_label: __param_target
   - target_label: __address__
+  {{- if eq . $.Values.global.region }}
     replacement: blackbox-exporter.kubernikus-system.svc:9115
+  {{- else }}
+    replacement: prober.{{ . }}.cloud.sap
```

Comment:
Do we want to use the prober from our bm clusters or do we want something like `prober.kubernikus.<region>.cloud.sap`?

Reply:
I am in favour of having as few blackbox-exporters as possible. It will get messy early enough and an overview of the infrastructure is already a bit daunting.
Right now, I would stick with the ones on the BM, especially as kubernikus is not rolled out everywhere yet.
If we want to move load away from the BM in the future, I would not change the URI in single configurations but rather let the DNS-entry for `prober.$region.cloud.sap` point somewhere else.

Reply:
Then it could be less daunting if we had the same setup in every cluster.

Reply:
I would like to propose to explicitly define a (list of) proper-urls in the values of the chart. That allows us to decouple this configuration from the SAP specific setup. Additionally, we don't need to generate the proper URL here. It also opens up possibilities for completely different setups, like having SaaS hosted probers or installations in a different cloud altogether:

```
blackbox-exporters:
  - http://pingdom.com/blackbox-exporter
  - https://prober.eu-de-1.cloud.sap
  - https://proper.na-us-1.cloud.sap
  - http://blackbox-exporter.kubernikus-system.svc:9115
```

One downside of that approach is that it also requires a loop (or it will hurt with dry-coding OCD). Another idea, would be to just have:

```
blackbox-exporters:
  primary: http://pingdom.com/blackbox-exporter
  secondary: https://prober.eu-de-1.cloud.sap
```

That would allow to simplify the config and wouldn't require a golang template loop at all. With the helm double evaluation that comes with it. Extra points because it will be easy to stick it into the `extra_scrape_config` and move the whole configuration out of the Prometheus chart. That gets us one step closer to being able to use the `stable/prometheus` chart.

Reply:
I'd even be happy with a single configurable prober...